indexer/src/types/transaction.ts

```diff
@@ -93,7 +94,7 @@ export function toEthTx({
     input: txJSON.data!,
     nonce: txJSON.nonce!,
     to: transaction.to?.toString() ?? null,
-    transactionIndex: isPendingBlock ? null : padBigint(BigInt(index ?? 0), 32),
+    transactionIndex: padBigint(BigInt(index ?? 0), 32),
```

Comment:
this stays null for a pending transaction

Reply:
same